tests/test_taskboard.py

```diff
@@ -0,0 +1,38 @@
+import taskboard
+import builtins
+
+class DummyClient(taskboard.TaskBoard):
+    def get_tasks(self, list_id: str):
+        return [{"id": f"{i}", "title": f"Task {i}"} for i in range(5)]
+
+    def add_task(self, list_id: str, title: str, **kwargs):
+        return "task-new"
+
+    def delete_task(self, task_id: str):
+        return True
+
+    def update_task(self, task_id: str, **kwargs):
+        return {"id": task_id, **kwargs}
+
+    def list_task_lists(self):
+        return [{"id": "inbox", "name": "Inbox"}]
+
+
+def test_main_taskboard(monkeypatch):
```

Comment:
can you make sure you add tests for all the methods?

Reply:
Added tests for all methods.